dapp-test/index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
```

Comment:
how has been written dapp-test? if it's not generated code, please consider not excluding it from prettier and eslint.

Reply:
its auto generated most of it. I have deliberately excluded it from eslint and prettier. See the npm commands.